curiouslearning/FeedTheMonsterJS
# Fix: Remove Next Level Icon after completing the Last Level of FTM (FM-281)

# Changes
- Fix and hide the next level icon after completing the last level of FTM

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Play the game until last level or toggle dev mode and select the last level and play
- Make sure next icon is not appearing in level end screen.
- Also test other languages just to make sure they work the same.

Ref: [FM-281](https://curiouslearning.atlassian.net/browse/FM-281)


[FM-281]: https://curiouslearning.atlassian.net/browse/FM-281?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
